plagiarism/http.go

```diff
@@ -0,0 +1,112 @@
+package plagiarism
+
+import (
+    "net/http"
+    "context"
+    "time"
+    "encoding/json"
+    "os/signal"
+    "os"
+    "log"
+)
+
+// Response struct will be used to transform to JSON format
+type Response struct {
+    Status string
+    Results map[string]string
+}
+
+// Server init variable
+type Server struct {
+    SubmissionDir string
+    ResultDir string
+    ID uint32
+    Logger *log.Logger
+    Port string
+}
+
+
+// Handler GET and POST in server
+func (s *Server) Handler() http.Handler {
+    return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+
+        // gluons only supports GET and POST reques to / path
+        // Throw HTTP error when path other than / is requested
+        if r.URL.Path != "/" {
```

Comment:
How will this work if there are multiple pending requests? Shouldn't the `GET` part also accept an ID? Does it make sense for the `POST` response to contain the ID?